ui/plugins/perf_details/main.html

```diff
@@ -1,17 +1,9 @@
-<div class="performance-panel">
-  <ul ng-show="perfJobDetail">
-    <li>Perfherder:
-      <ul ng-repeat="detail in perfJobDetail | toArray | orderBy: 'title'">
```

Comment:
I think the `toArray` here is the last usage of the `angular-toarrayfilter` package - so it can now be removed :-)

Reply:
Man, you are GOOD!  Thanks for catching this.  :)